ROOTFS/usr/bin/waggle-registration

```diff
@@ -5,28 +5,40 @@
 #  license.  For more details on the Waggle project, visit:
 #           http://www.wa8.gl
 # ANL:waggle-license
+import configparser
 import logging
 import os
 import os.path
 import re
 import subprocess
+import sys
 import time
 import hashlib
 from pathlib import Path
 
 
-logger = logging.getLogger('registration-service')
+formatter = logging.Formatter(
+    "%(asctime)s  [%(name)s:%(lineno)d] (%(levelname)s): %(message)s"
+)
+handler = logging.StreamHandler(stream=sys.stdout)
+handler.setFormatter(formatter)
+logger = logging.getLogger("registration-service")
 logger.setLevel(logging.INFO)
+logger.addHandler(handler)
 
-cert_server = 'beehive'
-registration_key = '/etc/waggle/id_rsa_waggle_registration'
+cert_server = "beehive"
```

Comment:
I see you're using black to stay consistent in formatting. I miss having a standard "go fmt" equivalent from Go. It's funny how a small choice like this eliminates whole classes of disagreements...

I was also starting to use [mypy](http://mypy-lang.org) and the typing module in a couple places to do some static analysis.